src/complate/index.js

```diff
@@ -19,15 +19,19 @@ export async function makeBundle(referenceDir) {
 }
 
 async function renderString(code, filename, context = {}) {
-	let { renderToString } = await load("rodunj/src/render", "complate extension");
+	let runtime = await load("complate-ast/dist/runtime", "complate extension");
+	let prefixed = {};
+	Object.keys(runtime).forEach(key => {
```

Comment:
Implementation note: I prefix the runtime dependencies of `complate-ast` to make collisions with user-defined identifiers less likely.